frontend/app/component/pollingstation/utils.ts

```diff
@@ -0,0 +1,23 @@
+export function getUrlForFormSectionID(baseUrl: string, id: string) {
```

Comment:
Needing to provide the base URL as a parameter feels silly, can we pass the election id and polling station id as parameters instead?